packages/block-library/src/media-text/style.scss

```diff
@@ -23,13 +23,25 @@
 	grid-area: media-text-content;
 	padding: 0 8% 0 8%;
 }
-.wp-block-media-text.has-media-on-the-right {
-	grid-template-areas: "media-text-content media-text-media";
-}
 
 .wp-block-media-text > figure > img,
 .wp-block-media-text > figure > video {
 	max-width: unset;
 	width: 100%;
 	margin-bottom: -6px;
 }
+
+@media (max-width: #{ ($break-small) }) {
```

Comment:
In general, we do the opposite. Build for mobile first and use `break-small` or `break-medium` to add the desktop styles. Isn't this possible here?

Reply:
No, in this case, we can't follow that approach because custom widths are set using an inline style, and on mobile, we need to overwrite this inline style so we need to use "!important". After having an important set on mobile styles it would be impossible than on the desktop sizes to make it use the inline styles again.
I should have added a comment with this explanation I will add them now.

Reply:
This is a good answer, but it's also a good question. Can you add an inline comment to explain basically the same?

Reply:
Hi @jasmussen I already added a CSS comment there specifying the reason for this, let me know if you find it hard to understand or you would prefer some improvement.

Reply:
Argh sorry, I missed that. No worries then, you're many steps ahead of me as usual.